controllers/app.controller.js

```diff
@@ -0,0 +1,45 @@
+const {
+  selectTopics,
+  selectArticleId,
+  updateArticleId,
+  selectUsers,
+  selectArticleComment,
+} = require("../models/app.model");
+
+exports.getTopics = (req, res) => {
+  selectTopics().then((results) => {
+    res.status(200).send({ results });
+  });
+};
+
+exports.getArticleId = (req, res) => {
+  const { article_id } = req.params;
+  selectArticleId(article_id).then((data) => {
+    res.status(200).send({ data });
+  });
+};
+
+exports.getArticleComment = (req, res) => {
```

Comment:
This should be pluralised, as it retrieves all of the comments for the given article, not just one.